rpm/SPECS/docker-ce.spec

```diff
@@ -34,15 +34,13 @@ BuildRequires: cmake
 BuildRequires: gcc
 BuildRequires: git
 BuildRequires: glibc-static
-BuildRequires: libarchive
```

Comment:
This one was added in https://github.com/docker/docker-ce-packaging/commit/78242140d7f85863e8a8e3a133f3ebf73b9596ca, as part of https://github.com/docker/docker-ce-packaging/pull/554

It was to fix a failure when compiling tini;

> According to https://bugs.centos.org/view.php?id=18212, upgrading to libarchive-3.3.3-1.el8.x86_64 should resolve the problem.

Looks indeed that it's no longer needed (or at least, CI is happy), but perhaps I should make this a separate commit that's a clean _revert_ of the other one to help correlate / track down history.